crates/cxx-qt-gen/test_inputs/properties.rs

```diff
@@ -11,6 +11,7 @@ mod ffi {
         #[derive(Default)]
         #[qproperty(i32, primitive)]
         #[qproperty(QPoint, trivial)]
+        #[qproperty(i32, test_name, read = my_getter, write = my_setter, notify)]
         type MyObject = super::MyObjectRust;
     }
```

Comment:
You also have some mixed cases that could go in here.

So you could have
```
#[qproperty(i32, custom_property, read = my_getter, write = my_setter, notify = my_notify)]
#[qproperty(i32, custom_partial_property, read = my_getter, write, notify = my_notify)]
```

Which would check that full custom generation works and also checks that eg the `my_notify` name is correctly chosen in the generated setter.

Unsure if you should add the methods/signals for the actual custom functions or not as they are in theory tested in other inputs and could make the output of this one even more verbose...

Reply:
done